README.md

```diff
@@ -1,163 +1,257 @@
-# Tower CLI app
+Nextflow Tower CLI
+==================
 
-Proof of concept of Tower command line client app based on Java native compilation, Micronaut HTTP declarative client and Tower domain classes
+`tw` is Nextflow Tower on the command line. It brings Tower concepts including Pipelines, Actions and Compute Environments to the terminal.
 
-## Installation
+![tw](/tw-screenshot.png)
 
-From the last Github action at artifacts section, download the binary for your OS. Unzip it, give it execution permissions with `chmod +x ./tw` and move it into a folder that it's in your path (ex: `sudo mv ./tw /usr/local/bin/tw`)
+Nextflow Tower is a full-stack application for the management of data pipelines and compute resources. It enables collaborative data analysis at scale, on-premises or in any cloud. 
 
-## Build binary development version
+The Tower CLI interacts with instances of Tower, providing an interface to launch pipelines, manage cloud resources and administer the application.
 
-1. Download GraalVM (Java 11 version) from [this link](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-20.2.0).
+The key features of the Tower CLI are:
 
-2. Install `native-image` tool:
+- **A Nextflow-like experience**: Tower CLI provides a developer friendly environment. Workflows can be launched similar to Nextflow but with the benefits of Tower such as monitoring, logging, resource provisioning, dataset management and collaborative sharing.
 
-    ```
-    gu install native-image
-    ``` 
+- **Infrastructure as Code**: All Tower resources including Workspaces and Compute Environments can be described in a declarative manner using YAML or JSON. This allows a complete definition of an analysis environment that can be versioned and treated as code. It greatly simplifies sharing, re-using and administration.
```

Comment:
Current version, the only resources that the CLI can import/export are launchpad pipelines and compute environments. And YAML is not supported, only JSON.